index.html

```diff
@@ -0,0 +1,754 @@
+                        <a href="#">
+                            <h3>Zwift companion app not working? Try this</h3>
+                        </a>
+                        <a href="#"><strong class="news__author">Matthew Beedham <span>9 hours ago</span></strong></a>
+                    </div>
+                </div>
+                <div class="latestItem">
+                    <img src="assets/images/news4.jpg" alt="Latest news 4">
+                    <div class="latestItem-title">
+                        <a href="#">
+                            <h3>Emissions have dropped 17% </h3>
+                        </a>
+                        <a href="#"><strong class="news__author">The Conversation <span>11 hours ago</span></strong></a>
+                    </div>
+                </div>
+                <div class="latestItem">
+                    <img src="assets/images/news5.jpg" alt="Latest news 5">
+                    <div class="latestItem-title">
+                        <a href="#">
+                            <h3>Satoshi Nakaboto: ‘Bitcoin continues downtrend with 5% loss in one day’ </h3>
+                        </a>
+                        <a href="#"><strong class="news__author">Satoshi Nakaboto <span>12 hours ago</span></strong></a>
+                    </div>
+                </div>
+                <div class="latestItem">
+                    <img src="assets/images/news6.png" alt="Latest news 6">
+                    <div class="latestItem-title">
+                        <a href="#">
+                            <h3>Adobe’s top tips to for your commerce</h3>
+                        </a>
+                        <a href="#"><strong class="news__author">Errol Denger <span>13 hours ago</span></strong></a>
+                    </div>
+                </div>
+                <div class="latestItem">
+                    <img src="assets/images/news7.jpg" alt="Latest news 7">
+                    <div class="latestItem-title">
+                        <a href="#">
+                            <h3>Space is full of junk because private companies keep launching satellites </h3>
+                        </a>
+                        <a href="#"><strong class="news__author">The Conversation <span>7 hours ago</span></strong></a>
+                    </div>
+                </div>
+                <div class="latestItem">
+                    <img src="assets/images/news6.png" alt="Latest news 8">
+                    <div class="latestItem-title">
+                        <a href="#">
+                            <h3> UK getting 200 mile route to test self-driving vehicles on public roads
+                            </h3>
+                        </a>
+                        <a href="#"><strong class="news__author">Matthew Beedham <span>13 hours ago</span></strong></a>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <section class="sponsor-section">
```

Comment:
### The `sponsor-section` and `article-section` is shrinking in width when height changes in desktop mode. This  is not the case in the original site

Right now 👇
![Annotation 2020-06-04 231152](https://user-images.githubusercontent.com/57726348/83792596-e5557e00-a6b8-11ea-899a-d44485e3fc08.jpg)

It should be 👇
![Annotation 2020-06-04 231220](https://user-images.githubusercontent.com/57726348/83792687-01591f80-a6b9-11ea-8c85-84a191b8f06d.jpg)